services/commissioning.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Rovercode Commissioning Service
+After=syslog.target docker.service
+Requires=docker.service
+
+[Service]
+TimeoutStartSec=0
+Restart=always
+User=root
+Group=root
+ExecStartPre=-/usr/bin/docker stop rovercode
```

Comment:
Do you mean to stop `rovercode-commissioning` here?

Reply:
Yup! Fixing.